### ISSUE TITLE
G5 search particle does not work in J4

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#3036]
Search particle is in many themes. It does not work in J4 and gives 0 Class "JEventDispatcher" not found

**Comment by mahagr** [comment]
Yes, as it is no longer shipped in Joomla 4. Replacement is com_finder.

PS. This issue is really not for Gantry itself, but RT themes such as Koleti. That said, I think we should move the search particle into Gantry itself.
